src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -63,7 +59,7 @@ def run_dm_dst_create(self, tool):
         pool1.connect(2)
 
         # Create a source cont
-        cont1 = self.create_cont(pool1, cont_type=self.cont_type)
+        cont1 = self.create_cont(pool1, cont_type)
```

Comment:
Careful here. cont_type is not the second parameter:
```suggestion
        cont1 = self.create_cont(pool1, cont_type=cont_type)
```
From:
https://github.com/daos-stack/daos/blob/c2d9edb10b11628f38317373a52ea40bd325cc9a/src/tests/ftest/util/data_mover_test_base.py#L269-L271